pages/reference/OS/updates/self-service.md

```diff
@@ -33,6 +33,10 @@ Update time can vary significantly, depending on the speed of your network, the
 
 If your {{ $names.os.lower }} update fails for any reason, the device should still be recoverable. For devices running {{ $names.os.lower }} 2.x, the user application should still be running normally, and the reasons for failure can be examined through the update logs at `/mnt/data/balenahup/` (or potentially the legacy location `/mnt/data/resinhup/`). If you have any issues, please contact us on the [troubleshooting section of the forums][troubleshooting].
 
+## Update locks
+
+[Update locks](https://docs.balena.io/learn/deploy/release-strategy/update-locking/) are mechanisms that enables a user service to run uninterrupted even if the Supervisor should be killing the service to perform an application update. This strategy now allows for users to signal the hostOS that it shouldn't reboot to complete a HostOS update. Some important steps during the update process will still happen, but since this reboot is required to fully update the OS, this will only happen once your update locks no longer exist. Overriding update locks (by means of the dashboard device configuration page) will still be respected in this case. Meaning, if you activate and set this configuration to true, a hostOS update will ignore the presence of your application update locks and will proceed with a reboot. Learn more about update locking [here](https://docs.balena.io/learn/deploy/release-strategy/update-locking/)
```

Comment:
A bit of rephrasing:

[Update locks](https://docs.balena.io/learn/deploy/release-strategy/update-locking/) is a mechanism that allows applications to enter critical sections and prevent updates that would interrupt the application from running.  Update locks can also be used to delay the reboot that applies a hostOS update operation until the application exists the critical section by removing the update locks. [Overriding update locks](https://docs.balena.io/learn/deploy/release-strategy/update-locking/#overriding-the-lock) will ignore existing locks and allow a hostOS update process to proceed with a reboot.